src/Pages/ShopPage/SingleProductPage/SingleProduct.js

```diff
@@ -29,44 +44,58 @@ const SingleProduct = (props) => {
                         </div>
                         <p className="green-txt bold">Inclusive of all taxes</p>
-                        {!productData.inStock && <p className="green-txt bold">Product Out Of Stock</p>}
+                        {!inStock && <p className="green-txt bold">Product Out Of Stock</p>}
                         <div>
-                            <p className="sm-p">Get it by Fri, Mar 04 - 600116</p>
-                            {productData.inStock && <p className="sm-p">Get it by Fri, Mar 04 - 600116</p>}
-                            <p className="sm-p">Seller: <span className="bold">{productData.seller.toUpperCase()}</span></p>
+                            {/* <p className="sm-p">Get it by Fri, Mar 04 - 600116</p> */}
+                            {inStock && <p className="sm-p">Get it by Fri, Mar 04 - 600116</p>}
+                            <p className="sm-p">Seller: <span className="bold">{seller.toUpperCase()}</span></p>
                         </div>
                         <div className="btn-section">
 
-                            <button className="btn warning-outlined-btn icon-left mg-t-10">
+                            <button onClick={() =>
+                                updateList({
+                                    product: productData.product,
+                                    state: wishlistState,
+                                    productFn: addToWishlist,
+                                    dispatchFn: wishlistDispatch,
+                                    dispatchType: 'UPDATE_WISHLIST'
+                                })}
+                                className="btn warning-outlined-btn icon-left mg-t-10">
                                 <i className="far fa-heart"></i>
                                 Add to WishList
                             </button>
-                            <button className="btn primary-btn solid icon-left mg-t-10">
+                            <button onClick={() => updateList({
+                                product: productData.product,
+                                state: cartState,
+                                productFn: addToCart,
+                                dispatchFn: cartDispatch,
+                                dispatchType: 'UPDATE_CART'
+                            })} className="btn primary-btn solid icon-left mg-t-10">
                                 <i className="fas fa-shopping-cart"></i>
                                 Add to Bag
                             </button>
                         </div>
                         <p className="sm-title mg-t-20">PRODUCT DESCRIPTION</p>
-                        <p className="rg-p">{productData.description}</p>
+                        <p className="rg-p">{description}</p>
                         <p className="sm-title mg-t-20">PRODUCT SPECS</p>
-                        <p className="rg-p">{productData.specs}</p>
+                        <p className="rg-p">{specs}</p>
```

Comment:
I think it would be better if PRODUCT DESCRIPTION and PRODUCT SPECS are replaced with Product Description and Product Specs for 2 reasons :

1. I personally find it hard to read all caps words, maybe this is just a preference and can be ignored.
2. I have read that maybe it is also bad for accessibilty.

https://github.com/humanmade/hm-pattern-library/issues/75#issuecomment-235672885

Reply:
woah okay, had no idea. Thank you for sharing the link!!